listenbrainz/webserver/static/js/src/user-feed/UserFeed.tsx

```diff
@@ -344,12 +352,8 @@ export default class UserFeedPage extends React.Component<
             )}
             showUsername={false}
             showTimestamp={false}
-            listen={metadata as Listen}
-            additionalContent={
-              (metadata as PinEventMetadata).blurb_content
-                ? `"${(metadata as PinEventMetadata).blurb_content}"`
-                : ""
-            }
+            listen={listen}
```

Comment:
@MonkeyDo The ListenCard uses the track_name and artist_name from the listen. However, if we have written a review for an artist, its likely we want to display the artist name first. (there's also some support for writing release group reviews so probably need to support that as well). Thoughts on how to do this? 

The entity_name and entity_type fields are the relevant ones here and should be used to display the name and link to the correct MB entity. For making the card playable, we can either just search the entity name and play it like in stats or use the attached recording_mbid mapped data to search for the original listen using which the review was written. Note that in the current implementation, I have attached recording mapped data in all cases with the listen in all cases.

To reiterate, the recording reviews are handled fine by the listen cards. The issue is how we support the display for other entity's reviews.

Reply:
I think I'm in favor of option # 1 and just using the entity name to search and play, like we do for the reports/charts pages ([example here](https://github.com/metabrainz/listenbrainz-server/blob/master/listenbrainz/webserver/static/js/src/stats/UserTopEntity.tsx#L203-L206)), but mainly for consistency.
I feel that if the user chose to review the artist rather than a specific recording we shouldn't tie a recording to their review on their behalf.

Reply:
Yes, given that I was arguing for consistency, I'm also good with #1